manifests/kustomize/env/gcp/kustomization.yaml

```diff
@@ -3,6 +3,7 @@ kind: Kustomization
 
 bases:
   - ../../namespaced
+  - ../gcp-dev/proxy
```

Comment:
nit: it's strange `/env/gcp` refer to `/env/gcp-dev`, because `/env/gcp` reads more general

maybe we can move proxy to `/env/gcp` and let `/env/gcp-dev` refer to it here

Reply:
it's already moved to in anther PR
https://github.com/kubeflow/pipelines/pull/3540